### DISCUSSION TITLE
Wanting to use command class directly.  Trying to import BinarySwitchCCAPI

### DISCUSSION THREAD

**Discussion by iklein99** [zwave-js/zwave-js#4511] [General] [answered]
I'm simply trying to use a command class directly.  I see that the class BinarySwitchCCAPI has the methods I'd need, but I can't import it.  When I use 'import { BinarySwitchCCAPI } from "zwave-js";' I get:
 '"zwave-js"' has no exported member named 'BinarySwitchCCAPI'. Did you mean 'BinarySwitchCCGet'?

I must be doing something wrong.  How to you access the get and set methods on this command class?

**Comment by AlCalzone** [comment, marked as answer]
You'll want to use something like

```
node.commandClasses["Binary Switch"].set(...)
```

where `node` is the node instance.

**Reply by iklein99** [reply]
Thanks!
